### ISSUE TITLE
Can I close tab in tab list interface?

### ISSUE THREAD

**Issue by TYZRPVX** [philc/vimium#2196]
After I typed Shift+t(T), I can search through tabs, but can I close tab in this interface using any shortcut ?

**Comment by JuanbiB** [comment]
Seems like you have to go to it, then 'x' to close it and then come back to where you where. 
I don't think there's a functionality to close the current tab your searching for. 

Maybe could be added? I'm p new to the Vimium ~world~

**Comment by leoj3n** [comment]
I like this idea. Seems like there could be a lot of opportunity for extensibility in the tab open interface. Would be nice if there was an API to interface with this part of Vimium to capture events and shuttle data to an auxiliary extension.

**Comment by smblott-github** [comment]
> Would be nice if there was an API to interface with this part of Vimium to capture events and shuttle data to an auxiliary extension.

Yes.  I'd like to see something like this.  #2109 already contains a mechanism for using link hints for picking a link and then passing it to an external extension.
